src/reactapp/src/components/checkoutAgreements/CheckoutAgreements.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import Card from '../common/Card';
+import Header from '../common/Header';
+import { __ } from '../../i18n';
+import useCartContext from '../../hook/useCartContext';
+import Checkbox from '../common/Form/Checkbox';
+
+function CheckoutAgreements() {
+  const {
+    getCheckoutAgreements,
+    checkoutAgreements,
+    changeCheckoutAgreement,
+  } = useCartContext();
```

Comment:
Instead of `useCartContext`, it will be better to create local hook (`useAgreementCartContext`) inside `checkoutAgreements/hooks` directory and use it everywhere